playbooks/roles/mail/handlers/main.yml

```
@@ -36,3 +36,9 @@
   service:
     name: postfix
     state: reloaded
+
+
+# Update the postfix header checks db
+# This should be run whenever /etc/postfix/header_checks is changed.
+- name: postmap transport
```

I think this should be `postmap header_checks`, the `postmap transport` handler already exists.

```suggestion
- name: postmap header_checks
```